extensions/amp-story/0.1/amp-story.css

```diff
@@ -14,5 +14,459 @@
+  justify-content: start;
+  padding: 32px;
+}
+
+.i-amphtml-story-grid-template-thirds {
+  grid-template-rows: repeat(3, 1fr) !important;
+  grid-template-areas: "upper-third"
+                       "middle-third"
+                       "lower-third" !important;
+  padding: 32px;
+}
+
+/** Bookend */
+.i-amphtml-story-bookend {
+  background: rgba(0, 0, 0, 0.85) !important;
+  color: #fff;
+  bottom: 0 !important;
+  display: none !important;
+  left: 0 !important;
+  position: absolute;
+  right: 0 !important;
+  top: 0 !important;
+  padding: 44px 32px 32px;
+  overflow: auto;
+}
+
+.i-amphtml-story-bookend-active .i-amphtml-story-bookend {
+  display: block !important;
+}
+
+.i-amphtml-story-bookend-heading {
+  text-transform: uppercase;
+  font-size: 10px;
+  padding: 0 0 8px;
+  margin: 48px 0 8px;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.25);
+  color: rgba(255, 255, 255, 0.54);
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  letter-spacing: 0.83px;
+  line-height: 1;
+}
+
+.i-amphtml-story-bookend-article-meta {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  font-size: 10px;
+  line-height: 1;
+  color: rgba(255, 255, 255, 0.54);
+}
+
+.i-amphtml-story-bookend-article-set > article {
+  margin: 24px 0;
+  overflow: hidden;
+}
+
+.i-amphtml-story-bookend-article-set:last-child > article:last-child {
+  margin-bottom: 0; /* bottom of bookend has padding > article margin */
+}
+
+.i-amphtml-story-bookend-article-heading {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  color: #ffff;
+  line-height: 1.4;
+  margin: 0 0 8px;
+}
+
+.i-amphtml-story-bookend-article-image {
+  width: 116px;
+  height: 116px;
+  float: right;
+  margin-left: 16px;
+}
+
+.i-amphtml-story-bookend-heading + .i-amphtml-story-bookend-article-set {
+  border-top: none;
+}
+
+.i-amphtml-story-bookend-article-set {
+  border-top: 1px solid rgba(255, 255, 255, 0.25);
+}
+
+.i-amphtml-story-share-icon > svg {
+  fill: #fff;
+}
+
+.i-amphtml-story-share-list {
+  display: flex;
+  list-style: none;
+  flex-wrap: nowrap;
+  padding: 16px 0;
+  /* negative margin = outer padding for container for horizontal scrolling */
+  margin: 0 -32px;
+  overflow: auto;
+}
+
+.i-amphtml-story-share-list::-webkit-scrollbar {
+  width: 0px;
+  background: transparent;
+}
+
+.i-amphtml-story-share-icon {
+  box-sizing: border-box;
+  width: 48px;
+  height: 48px;
+  padding: 8px;
+  background-color: rgba(255, 255, 255, 0.3);
+  border-radius: 4px;
+  background-size: 32px 32px;
+  display: block !important;
+}
+
+.i-amphtml-story-share-icon:active {
+  background-color: rgba(255, 255, 255, 0.6);
+  outline: none;
+}
+
+amp-social-share.i-amphtml-story-share-icon[type=email] {
+  background-image: none !important;
+}
+
+.i-amphtml-story-share-list > li {
+  width: 48px;
+  height: 66px;
+  padding: 0 16px;
+  display: inline-block;
+}
+
+.i-amphtml-story-share-list > li:first-child {
+  padding-left: 32px;
```

Comment:
dittor, this and below need rtl rules to reset left to 0 and apply 32px to right